plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version4/Products/class-wc-rest-products-v4-controller-tests.php

```
@@ -0,0 +1,1994 @@
+		$this->assertEquals( $response_products[1]['sku'], 'waffle-3' );
+
+		remove_filter( 'wc_product_sku_enabled', '__return_false' );
+	}
+
+	/**
+	 * Test that the products endpoint can filter by global_unique_id.
+	 *
+	 * @return void
+	 */
+	public function test_products_query_by_global_unique_id_param() {
+		$product = WC_Helper_Product::create_simple_product(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$product'. (Unused Code Rules)

(UnusedLocalVariable)